.github/workflows/master.yml

```diff
@@ -51,6 +51,17 @@ jobs:
       NEXT_PUBLIC_UNLOCK_GA_ID: ${{ secrets.UNLOCK_PROTOCOL_COM_NETLIFY_PROD_UNLOCK_GA_ID }}
       NEXT_PUBLIC_UNLOCK_ENV: prod
 
+  deploy-docs-production:
+    if: ${{ github.repository_owner == 'unlock-protocol' }} # make sure we dont deploy from forks
+    uses: ./.github/workflows/_netlify.yml
+    with:
+      bypass_diff_check: bypass
+      service: docs
+      target-env: prod
+    secrets:
+      SITE_ID: d6588eac-a04f-43dc-8341-f85d830bff49
+      AUTH_TOKEN: ${{ secrets.UNLOCK_PROTOCOL_COM_NETLIFY_PROD_AUTH_TOKEN }}
```

Comment:
Where did you set that one? Would you please add to 1password as well so we can easily get secrets from there?

Reply:
It is already set. It is the default netlify auth token. I'm not sure why we duplicate it though.